content/docs/infrastructure-wg.md

```diff
@@ -0,0 +1,196 @@
+   dev meeting (in April 2021).
+
+## Proposal for day-to-day work mode
+
+This section describes a proposal for the flow for the day to day work of the 
+working group. Note that only the good case is described here. A proposal can be
+cancelled in any of these steps. There might also be some iterations as some 
+parts of a statement of work (SOW) are not clear or feasible.
+
+1. Anyone in the LLVM community (including contractors) can propose a change by 
+   creating an issue on the issue tracker. 
+1. The working group will create an statement of work (SOW) based on the  
+   proposals they want to move forward on. The SOW will consist of a problem 
+   description and (functional and non-functional) requirements for a solution 
+   of the problem. The SOW shall be made public and discussed with the 
+   community.
+1. Once the SOW and it’s costs are clear enough, the working group will decide 
+   if and how it shall be implemented (community contribution, paid community 
+   bounty, contractor)
+    1. If it requires spending money, the working group will propose the change 
+       to the board.
+    1. The board will then review and approve the expenditure.
+    1. The Foundation will set up the contract and manage the payments.
+1. The working group will work with the contractor/community on a day-to-day 
+   basis to get the work done.
+1. The working group will approve the work and authorize payments with the 
+   treasurer (if needed).
+
+All of this shall be open and transparent for the community. Feedback from the
+community is appreciated in all of these steps. In parallel to this, community 
+members are welcome to take over certain SOWs and will get help from the 
+respective contractor if needed. If that work touches shared infrastructure or 
+impacts the future maintenance, the solution needs to be agreed with the 
+working group and the contractor.
+
+## Proposal for implementation and operations
+
+To keep the workload for the IWG low and scale up the work, the implementation
+of changes shall be mostly done by a 3rd party. 
+
+*Operations of services* shall be done by contractors with whom we can have
+Service Level Agreements (availibility, response times, recovery time, 
+handling GDPR requests, ...).
+
+For *implementing individual changes* there are a few options:
+* Community volunteers: There is someone from the community willing to 
+  implement the change.
+* Community bounty: We're offering a bounty for a community member to implement
+  the change. This is a chance to give back something to community members
+  who are not paid for their contirbutions by some company or organisation.
+* Contractor: In case something is very large or we can't find anyone to do it
+  the work can be handed off to a contractor.
+
+
+## Ideas for initial work items
+
+This is an unsorted, unprioritized list of ideas the working group could look
+into: 
+
+* Run a survey in the community to understand the largest pain points and 
```

Comment:
Yes! I don't really have a clear picture on what the questions we should ask, but I would like to identify the major pain points of the community and get them fixed (or less painful). This would be a nice showcase for the work of the IWG and to gain trust in the community and foundation for larger topics.